### PR TITLE
juan-tex adds Edinburgh, UK

### PR DESCRIPTION
**Pull request by juan-tex**
This pull request is to add Edinburgh, UK, to the list of hometowns. 

@mpvelarde @davidtextensor Check it out :D

### CODE REVIEW

File: juan-tex-edinburgh.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Edinburgh, UK
+
+### Resturants
+
+- Piatto Verde
+- Domenicos
```
Comment (sighte) [inline juan-tex-edinburgh.md]: Is this even better than Dominos?

Comment (juan-tex) [inline juan-tex-edinburgh.md]: Hahaha, I don't know. They've just been recommended to me by a food critic, so I'm looking forward to trying them :D

**Comment by juan-tex** [comment]
so, I added La Favorita, which is definitely better than Domino's. @sighte can I get an approval?

**Comment by sighte** [comment]
> so, I added La Favorita, which is definitely better than Domino's. @sighte can I get an approval?

Looking good 😄

**Pull request merged**